src/pages/ProductDetail/ProductHeadSection.js

```diff
@@ -0,0 +1,225 @@
+import React from 'react';
+import styled from 'styled-components';
+import Slider from 'react-slick';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+
+const ProductHeadSection = ({ product }) => {
+  var settings = {
+    dots: true,
+    infinite: true,
+    speed: 500,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+  };
+  return (
+    <HeadSection>
+      <HeadImg className="wrapper">
+        <Slider {...settings} className="imgSlider">
+          {product.image &&
+            product.image.map(image => (
+              <div>
+                <div>
+                  <img alt="상품 사진" src={image} />
+                </div>
+              </div>
+            ))}
+          {/* <div>
+            <div>
+              <img alt="상품 사진" src={product.image && product.image[0]} />
+            </div>
+          </div>
+          <div>
+            <div>
+              <img alt="빵사진" src={product.image && product.image[1]} />
+            </div>
+          </div>
+          <div>
+            <div>3333333333</div>
+          </div> */}
+        </Slider>
+      </HeadImg>
+      <HeadInfo>
+        <InfoWrap>
+          <InfoTitle>{product && product.title}</InfoTitle>
+          <InfoPrice>
+            {product.discount_percent !== 0 && (
+              <PricePercent>
+                {product && product.discount_percent}%
+              </PricePercent>
+            )}
+            {product.discount_percent !== 0 && (
+              <PriceDiscount>
+                {product.price && product.discounted_price.toLocaleString()}원
+              </PriceDiscount>
+            )}
+            <PriceOriginal>
+              {product.price && product.price.toLocaleString()}원
+            </PriceOriginal>
+          </InfoPrice>
```

Comment:
```suggestion
          <InfoPrice>
            {product.discount_percent !== 0 && (
              <>
                <PricePercent>
                  {product && product.discount_percent}%
                </PricePercent>
                <PriceDiscount>
                  {product.price && product.discounted_price.toLocaleString()}원
                </PriceDiscount>
              </>
            )}
            <PriceOriginal>
              {product.price && product.price.toLocaleString()}원
            </PriceOriginal>
          </InfoPrice>
```
중복되는 조건을 하나로 합칠 수 있습니다!